README.md

```diff
@@ -91,9 +91,9 @@ We will create an app that simply reads and writes a value to the blockchain. We
 
 Now it is time to create a solidity smart contract.
 
-1. You can use any editor you like to make the contract. For this tutorial we recommend the online IDE [remix.ethereum.org]
+1. You can use any editor you like to make the contract. For this tutorial we recommend the online IDE [https://remix.ethereum.org]
```

Comment:
Might be worth it to just make this a Markdown link as well. Something like

`[Remix](https://remix.ethereum.org)`